pkg/agent/dozer/bcm/plan.go

```diff
@@ -1034,6 +1031,31 @@ func planExternals(agent *agentapi.Agent, spec *dozer.Spec) error {
 				},
 			},
 		}
+
+		irbVLAN := agent.Spec.Catalog.IRBVLANs[externalName]
+		extVNI := agent.Spec.Catalog.ExternalVNIs[externalName]
+		if irbVLAN == 0 { //nolint:gocritic
+			// TODO: make this an error eventually, but not now to allow agent updates
+			slog.Warn("IRB VLAN for external not found in catalog, not configuring it", "external", externalName)
```

Comment:
I think it should return an error already as controller always updated before the agent

Reply:
not happening anymore with the new changes